packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -62,6 +62,9 @@ enum SelectionChangedCause {
   /// The user used the mouse to change the selection by dragging over a piece
   /// of text.
   drag,
+
+  /// The user iPadOS 14 Scribble to change the selection.
```

Comment:
Perhaps, but I don't know the behavior for those – for scribble specifically, this is needed so that we show the selection handles like native the iPadOS UITextField does when using scribble to change the selection...I don't know if that is the case for using a stylus to change the selection on android, for example